Assets/Scripts/Interaction/Hand.cs

```diff
@@ -155,8 +180,160 @@ public void setLock(bool value)
     }
 
     public void chargeSword(bool charge) {
-        if (charge) sword.StartCoroutine("swordCharge");
-        else sword.StopCoroutine("swordCharge");
+        if (charge) {
```

Comment:
I had to add a stop sound here to ensure that the charge sound wouldn't go if the user stopped charging